### PR TITLE
Rahul/code cleanup

### PR DESCRIPTION
**Pull request by rparundekar**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "License :: OSI Approved :: MIT License",
+]
+
+[tool.pytest.ini_options]
+addopts = "-vvv"
+testpaths = "tests"
```
Comment (shankarg87) [inline pyproject.toml:20]: we don't have any test paths yet right?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+[project]
+name = "Fine-Tuning Reserch"
+version = "0.1.0"
+description = "Open source research on fine-tuning LLMs"
+authors = ["Rahul Parundekar <rahul@aihero.studio>", "Shankar Ganesan <gshankar.87@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://aihero.studio"
+repository = "https://github.com/ai-hero/python-project-template"
```
Comment (shankarg87) [inline pyproject.toml:9]: is this repository correct?

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+[project]
+name = "Fine-Tuning Reserch"
```
Comment (shankarg87) [inline pyproject.toml:2]: Is it acceptable to have a space in the name? would be better to rename this as "fine-tune-research" so the package can (eventually) be downloaded through pip?

File: pyproject.toml (hunk 1 above)
Comment (rparundekar) [reply pyproject.toml:20]: Yea, but let's keep this here when we add tests (I cant sem to comment it out)

File: pyproject.toml (hunk 2 above)
Comment (rparundekar) [reply pyproject.toml]: Fixed

File: pyproject.toml (hunk 3 above)
Comment (rparundekar) [reply pyproject.toml:2]: Yup.

Comment (rparundekar) [reply pyproject.toml:2]: Oh wait the name here is pip compatible name?

Comment (shankarg87) [reply pyproject.toml:2]: not clear. I think they maybe different - but would still go with ```fine-tune-research```

**Review by shankarg87 — APPROVED** [review]

**Pull request merged**